frontend/src/admin/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from "react";
+import DropdownMenuIcon from "../dropdown-menu-icon/dropdown-menu-icon";
+import "./dropdown-menu.scss";
+
+interface DropdownMenuProps{
+    items: string[]
+    onClick: (value: string) => void
+}
+
+export default function DropdownMenu( {items, onClick}: DropdownMenuProps) {
+
+    const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+    const toggleDropdown = () => {
+        setShowDropdown(!showDropdown);
+    };
+
+    const hideDropdown = (event: React.FocusEvent<HTMLButtonElement>): void => {
+        if (event.currentTarget === event.target) {
+            setShowDropdown(false);
+        }
+    };
+
+    return (
+        <div className={showDropdown ?"c-dropdown-menu--active" : "c-dropdown-menu"}>
+            <button
+                onClick={() => toggleDropdown()}
```

Comment:
As `toggleDropdown` doesn't take any arguments, and return void, the line can be written as suggested.

```suggestion
                onClick={toggleDropdown}
```

Reply:
Cool thanks, this is implemented ✅